host/tests/dsCompositeIn_L3_Tests/dsCompositeIn_L3_testSetup.yml

```diff
@@ -1,20 +1,30 @@
 # this file is generated by the testing suites, when a new test is modified, this configuration file will be filled out and updated.
 #location: https://github.com/rdkcentral/rdk-halif-test-device_settings/tree/3.1.2/assets/
 
-dsCompositeIn:  # Prefix must always exist
+dsCompositeIn:
   description: "dsCompositeIn Device Settings test setup"
   assets:
     device:
-      Common: #List of common requirements for all the tests
+      defaults: &defaults
         artifacts:
-          -  "<URL>/hal_test" #URL Path to the bin files to copy
-          -  "<URL>/ut_control.so" #URL Path to the .so files if any to copy
-          -  "<URL>/run.sh"
+          - "http://localhost:8000/bin/hal_test"
+          - "http://localhost:8000/bin/libut_control.so"
```

Comment:
remove the hard coded url. keep like <URL>/bin/hal_test

Reply:
done